go.mod

```diff
@@ -105,3 +105,7 @@ require (
 	google.golang.org/grpc v1.58.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/livekit/mediatransportutil => ../mediatransportutil
+
+replace github.com/livekit/protocol => ../protocol
```

Comment:
Will fix these up once the corresponding PRs are approved. Making some small changes in those repos to mark marshallable fields as yaml and add `omitempty` everywhere.